src/base-container/index.scss

```diff
@@ -1,7 +1,11 @@
 @import "~@edx/brand-edx.org/paragon/variables";
 
 .authn-component__modal {
-  max-height: 98vh !important;
+  max-height: 98vh;
```

Comment:
Would this be applied in the prospectus without important? We need to confirm it.